test/arbitrum/HimalayaCompoundArbitrum.t.sol

```diff
@@ -149,6 +165,7 @@ contract HimalayaCompoundArbitrumUnitTests is HimalayaCompoundUtils, ConnextUtil
   }
 
   function test_handleInboundToV3WithHimalayaConnextWithInvalidAmount() public {
+    // TODO this test must be updated with the expected behavior of try-catch
```

Comment:
try catch will affect this test

Reply:
this comment applies to all the three test files:
- arbitrum, mainnet, and polygon